pkg/nsx/client.go

```diff
@@ -131,3 +140,28 @@ func (client *Client) NSXCheckVersionForSecurityPolicy() bool {
 	client.NSXVerChecker.securityPolicySupported = true
 	return true
 }
+
+func (client *Client) NSXCheckVersionForStaticRoute() bool {
```

Comment:
can we make this method common instead of static route only, since most vpc resources start from this nsx version.